### PR TITLE
Refactor/modal animation

### LINKED ISSUE #31: Refactor styles modal create channel
- [x] Add animation when the modal is showing.
- [x] Clean input when channel is created.
- [x] Render new channel in list channel.

### PR DESCRIPTION
**Pull request by byteconde**
### This pull request includes:

* Display model when add button is pressed
* Add animation with keyframes to modal window

closed #31

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -169,3 +169,61 @@ a {
...
+.dialog::backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, .5);
+  animation: none;
+}
+.dialog[open]::backdrop {
```
Comment (condef5) [inline public/css/style.css]: Hey you could add blank spaces between lines :).

File: public/index.html [hunk 2]
```diff
@@ -27,9 +27,14 @@ <h4 class="user__name" id="user__name">Diego Cuevas</h4>
   </div>
   <div class="create-channel p-10">
     <p>Create channel</p>
-    <svg class="icon" id="create">
-      <use xlink:href="#icon-plus">
-    </svg>
+    <!-- Button add channel -->
+
+    <button id="show">
+      <svg class="icon" id="create">
```
Comment (condef5) [inline public/index.html]: Why are there two ids?

File: public/index.html [hunk 3]
```diff
@@ -27,9 +27,14 @@ <h4 class="user__name" id="user__name">Diego Cuevas</h4>
   </div>
   <div class="create-channel p-10">
     <p>Create channel</p>
-    <svg class="icon" id="create">
-      <use xlink:href="#icon-plus">
-    </svg>
+    <!-- Button add channel -->
+
+    <button id="show">
```
Comment (condef5) [inline public/index.html]: Thanks for adding a button, that improves semantics :)

File: public/css/style.css (hunk 1 above)
Comment (byteconde) [reply public/css/style.css]: Yes, I'm working on it, thanks

**Comment by condef5** [comment]
Good job bro, thank you for your effort.

File: public/index.html (hunk 2 above)
Comment (byteconde) [reply public/index.html]: I will fix it a.s.a.p, thanks Frank

File: public/index.html (hunk 3 above)
Comment (byteconde) [reply public/index.html]: You're welcome dude

File: public/index.html [hunk 4]
```diff
@@ -27,9 +27,14 @@ <h4 class="user__name" id="user__name">Diego Cuevas</h4>
   </div>
   <div class="create-channel p-10">
     <p>Create channel</p>
-    <svg class="icon" id="create">
-      <use xlink:href="#icon-plus">
-    </svg>
+    <!-- Button add channel -->
+
```
Comment (DiegoCuevas) [inline public/index.html]: Be careful with the extra spaces. By the way good job. :+1:

**Pull request merged**